data/campaigns/Heir_To_The_Throne/scenarios/11_Elensefar_After.cfg

```diff
@@ -0,0 +1,709 @@
+                [/message]
+                {MOVE_UNIT id=Konrad $coordX $coordY}
+            )}
+        [/event]
+
+        #############################
+        # ALL LEADERS KILLED
+        #############################
+        [event]
+            name=last breath
+            {FILTER id=foxtail1,foxtail2}
+            [message]
+                speaker=unit
+                message=_"I’m not dying! Orcs never die! I just... need a moment..."
+            [/message]
+            [event]
+                name=die
+                {UNSTORE_NPC Reglok x,y=25,16 side,facing=5,se}
+                {GENERIC_UNIT 5 Thief 27 16} {ANIMATE} {GENDER female} {ROLE thief}
+                [message]
+                    speaker=Reglok
+                    message=_"Well done, well done! Elensefar is finally rid of those loathsome beasts."
+                [/message]
+                [message]
+                    speaker=Konrad
+                    message=_"You walk free once more! But for how long? Once we leave, who will remain to defend the bridges?"
+                [/message]
+                [message]
+                    speaker=Reglok
+                    message=_"They say the best defense is a good offense, no? I place the thieves of Elensefar in your service, your exaltedness, ready to take the fight to the dark queen."
+                [/message]
+                {NEW_RECRUIT2 (Thief) _"Konrad can now recruit Thieves!" human-outlaws/thief.png human-outlaws/thief+female.png}
+                [message]
+                    speaker=Reglok
+                    message=_"I myself must of course remain behind to oversee the city. You understand."
+                [/message]
+                [message]
+                    speaker=Konrad
+                    message=_"All too well. Rule wisely, ambitious Reglok, for I shall return to Elensefar once Asheviere is defeated and I am king. Prove yourself a better lord than the orcs."
+                [/message]
+
+                {CLEAR_VARIABLE s11_from_north,thieves_will_ambush,defender_deaths}
+                [endlevel]
+                    result=victory
+                    {NEW_GOLD_CARRYOVER 40}
+                [/endlevel]
+            [/event]
+        [/event]
+    [/event]
+
+    #############################
+    # TIME OVER
+    #############################
+    [event]
+        name=turn $({SCENARIO_TURN_LIMIT}-4)
+        [message]
+            speaker=Konrad
+            message=_"Between the siege and our attack, Elensefar has been badly damaged. Countless townsfolk will die if we cannot finish things off soon!"
+        [/message]
+    [/event]
+    [event]
+        name=side 1 turn {SCENARIO_TURN_LIMIT} end
+        [message]
+            speaker=Konrad
+            message=_"Our battle has dragged on too long; Elensefar has become too badly destroyed. Even if we defeat these orcs, the city will never-again return to its former splendor..."
```

Comment:
Suggest either 'Elensefar has become too badly damaged' or 'Elensefar is all but destroyed'.